content/news/XXXX-XX-XX-new_icon_contest.md

```diff
@@ -0,0 +1,76 @@
+title: Contest for a new updated Mixxx Icon
+authors: Jörg Wartenberg
+tags: icon, logo
+date: 2025-02-16 10:00:00
+
+## Why Mixxx needs a new updated icon
+The square Mixxx Icon was created at a time when it was only used to identify the application in the desktop environment of the respective operating system. For this use case, it had to be displayable on low resolution at 16x16px.
+![Mixxx TitleBar]({static}/images/news/icon-contest/Mixxx_Icon_in_titlebar.png)
+It was designed with a specific, outdated theme guideline in mind and always looked odd on all other systems.
+![Mixxx Icon]({static}/images/news/icon-contest/mixxx_icon.svg)
+Nowadays, square app icons are no longer only found on the desktop. With the advent of online app stores, the square icon has become the primary identifier for the application. A square icon with a much higher resolution (typical 256x256px) is required here. In the stores we cannot use our rectangular Mixxx Logo due to the low aspect ratio there and since our square icon cannot be easily adapted to the required style guidelines. For example, Mixxx has not met the criteria to be listed as a curated app on Flathub and will therefore always apear on the lower ranks.
```

Comment:
```suggestion
With the advent of online app stores, the square icon has become the primary identifier for the application. A square icon with a much higher resolution (typical 256x256px) is required here. In the stores we cannot use our rectangular Mixxx Logo due to the low aspect ratio there and since our square icon cannot be easily adapted to the required style guidelines. For example, Mixxx has not met the criteria to be listed as a curated app on Flathub and will therefore always apear on the lower ranks.
```